crates/solidity/inputs/language/src/definition.rs

```diff
@@ -3141,34 +3146,69 @@ codegen_language_macros::compile!(Language(
                                 PrecedenceExpression(
                                     name = AssignmentExpression,
                                     rule_name = BinaryExpression,
-                                    operators = [PrecedenceOperator(
-                                        model = BinaryLeftAssociative,
-                                        fields = (operator = Required(Terminal([
-                                            Equal,
-                                            BarEqual,
-                                            PlusEqual,
-                                            MinusEqual,
-                                            CaretEqual,
-                                            SlashEqual,
-                                            PercentEqual,
-                                            AsteriskEqual,
-                                            AmpersandEqual,
-                                            LessThanLessThanEqual,
-                                            GreaterThanGreaterThanEqual,
-                                            GreaterThanGreaterThanGreaterThanEqual
-                                        ])))
-                                    )]
+                                    operators = [
+                                        PrecedenceOperator(
+                                            model = BinaryLeftAssociative,
```

Comment:
Do you mean `UsingDeconstructionSymbol`? `UsingOperator` was introduced in this PR IIUC and only used in `UsingAlias`.

I agree with the unnecessary wrapping argument though. The win in this PR was to remove the redundant `Required`/`Optional` elsewhere but in cases like these we're replacing 15 lines with 50. Do you think we could simplify this somehow in the future?

Reply:
My bad. I mixed the two. I think this change made the syntax more coincise in the 98% of the cases, and worth it to sacrifice a few extra lines in the remaining 2%. As far as the output is concerned, it will eventually be identical (once migration is complete).

But happy to reconsider if you feel otherwise.